test/IDE/complete_call_arg.swift

```diff
@@ -1271,6 +1271,7 @@ func testSkippedCallArgInInvalidResultBuilderBody() {
     Other(action: 2) {
       MyImage(systemName: "", #^INVALID_RESULTBUILDER_ARG^#
     struct Invalid
+    }
```

Comment:
This missing `}` caused a test failure because `extension` is not allowed in a local context.